.vscode/launch.json

```diff
@@ -56,6 +66,16 @@
             "internalConsoleOptions": "neverOpen",
             "cwd": "${workspaceFolder}/packages/fiori-freestyle-writer"
         },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "fiori-elements-writer: Debug Current Jest File",
+            "program": "${workspaceFolder}/node_modules/jest/bin/jest.js",
+            "args": ["${file}", "--config", "jest.config.js", "--coverage=false"],
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "cwd": "${workspaceFolder}/packages/fiori-elements-writer"
+        },
```

Comment:
Do we still need these configurations? The instructions here work well: https://github.com/SAP/open-ux-tools/blob/c77e420bf59ad83bef809b44bfc1e0690da99052/README.md?plain=1#L96

Reply:
I think it is more convenient tbh.